### ISSUE TITLE
[Bug] Clicking on Light/Dark mode theme causes `Keyframe rule ignored due to bad selector` error

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/TwitchBot#87]
Navigated to any page, click on the light mode / dark mode switch in the top nav from any state, light or dark.  This happens on any route, and in any state either dark or light.

Clicking on the button causes a `Keyframe rule ignored due to bad selector.`

**Comment by Sky-De** [comment]
@gbowne1 hi, I was checking it, and I did not encounter the issue that you have mentioned, I provide a GIF as a showcase

### GIF ( wait for load )

![chrome-capture-2024-4-14](https://github.com/gbowne1/TwitchBot/assets/79264045/5c11e3aa-adad-4c66-81fb-15310ea3674e)



**May I ask you to provide some visual examples of your issue like an image or GIF?**

**Comment by gbowne1** [comment]
@Sky-De 

Did you look at the browser console or devtools to see if there was a keyframe error/warning while toggling the dark / light mode?

**Comment by Sky-De** [comment]
> @Sky-De
> 
> Did you look at the browser console or devtools to see if there was a keyframe error/warning while toggling the dark / light mode?

@gbowne1 yes I did, but there was no such message that you had mentioned, I will attach a video of it

**Comment by Sky-De** [comment]
### Video ( https://github.com/gbowne1/TwitchBot/issues/87#issuecomment-2110751539 )
https://github.com/gbowne1/TwitchBot/assets/79264045/fccfeedd-64d2-4528-b4ff-61c7a1be762c

**Comment by gbowne1** [comment]
Ok thanks @Sky-De
